### PR TITLE
CHAOS-5241 Update prerequisites.md

### PR DESCRIPTION
**Pull request by SmritiSatya**
Update prerequisites.md

### CODE REVIEW

File: docs/chaos-engineering/get-started/tutorials/prerequisites.md [hunk 1]
```diff
@@ -6,7 +6,18 @@ redirect_from:
...
 ---
 
-This section describes the requirements, such as chaos infrastructure and project-level permissions required to execute chaos experiments on your target environments.
+This section describes the requirements and project-level permissions required to execute chaos experiments on your target environments.
+
+- Prepare yourself with the right permissions (`Chaos Resources Role Permissions` in `Access Control`).
+- Prepare yourself with the right permissions on the cloud account or the Kubernetes cluster or the VM (Kube RBAC, IAM Roles).
+- Enable the necessary Feature Flags (as a general step) and corresponding sanity checks (such as places to click, and entities to see enabled).
+- Prepare the target systems (VMs or K8s).
+- Prepare **network connectivity**, identify **proxy requirements**, **firewall rules** (if any).
```
Comment (douglas-j-bothwell) [inline docs/chaos-engineering/get-started/tutorials/prerequisites.md:15]: Make these links?

File: docs/chaos-engineering/get-started/tutorials/prerequisites.md [hunk 2]
```diff
@@ -6,7 +6,18 @@ redirect_from:
...
+- Identify application (or infrastructure) steady state parameters (even if this requires manual effort)- using APMs or logs or other methods.
```
Comment (douglas-j-bothwell) [inline docs/chaos-engineering/get-started/tutorials/prerequisites.md:16]:
```suggestion
- Identify application (or infrastructure) steady-state parameters (even if this requires manual effort)- using APMs or logs or other methods.
```

File: docs/chaos-engineering/get-started/tutorials/prerequisites.md [hunk 3]
```diff
@@ -6,7 +6,18 @@ redirect_from:
...
+- Identify image registry requirements and steps to set it up with secrets.
```
Comment (douglas-j-bothwell) [inline docs/chaos-engineering/get-started/tutorials/prerequisites.md:17]:
```suggestion
- Identify image registry requirements and steps to set up the registry with secrets.
```

File: docs/chaos-engineering/get-started/tutorials/prerequisites.md [hunk 4]
```diff
@@ -6,7 +6,18 @@ redirect_from:
...
+- Identify specific needs, especially Kubernetes, such as namespace quotas, workloads should carry specific labels or annotations, workloads should carry resource limits, every outbound container needs proxy environments, workloads should reside in specific nodes or groups.
```
Comment (douglas-j-bothwell) [inline docs/chaos-engineering/get-started/tutorials/prerequisites.md:18]:
```suggestion
- Identify specific needs, especially for Kubernetes. You might need to specify namespace quotas, workload-specific labels or annotations, workload resource limits, proxy environments for outbound containers, and the specific nodes or groups where workloads should reside.
```
Also consider breaking these out into a list of sub-bullets

File: docs/chaos-engineering/get-started/tutorials/prerequisites.md [hunk 5]
```diff
@@ -6,7 +6,18 @@ redirect_from:
...
+- Identify permissions for advanced use cases, which may vary, such as SCC, IRSA, etc.
+- ChaosHub requirement and connectivity to Git sources.
```
Comment (douglas-j-bothwell) [inline docs/chaos-engineering/get-started/tutorials/prerequisites.md:20]:
```suggestion
- ChaosHub requirements and connectivity to Git sources.
```

**Review by douglas-j-bothwell — DISMISSED** [review]
A few sugs but otherwise LGTM

**Review by neelanjan00 — APPROVED** [review]

**Pull request merged**